session-keys/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+authors = ['Manta Network']
+description = "Primitives for session keys"
+edition = "2021"
+name = "session-key-primitives"
+version = "0.1.0"
```

Comment:
license.
And the version is `0.1.0`, kind of misaligned with other crates.

Reply:
thanks